app/uk/gov/hmrc/sdes/bulkdownload/controllers/BulkDownloadController.scala

```diff
@@ -42,7 +42,7 @@ class BulkDownloadController @Inject()(sdesListFilesConnector: SdesListFilesConn
     Logger.debug(s"request headers from /list: ${request.headers.toSimpleMap}")
     Logger.debug(s"HeaderCarrier headers from /list: ${hc.headers}")
     sdesListFilesConnector.listAvailableFiles(fileType)(hc) map {
-      case Nil => NotFound
+      case Nil => Ok(JsArray())
       case nonEmptyList => Ok(Json.toJson(nonEmptyList))
     } recover {
```

Comment:
Optional sugestion: I propose to return just a list as it is from search results, like
```Scala
sdesListFilesConnector.listAvailableFiles(fileType)(hc)
  .map(list => Ok(Json.toJson(list)))
  .recover {
    ...
  }
```

Reply:
incorporated all the comments